### PR TITLE
Use `react-lowlight` to highlight and `json-beautify`, `xml-formatter` to prettify the `EntryBodySection`

### PR DESCRIPTION
**Pull request by mertyildiran**
Also makes the Base64 decoding optional but makes it `true` by default.

### Before

Screenshot from 2021-12-26 02-54-02

### After

Screenshot from 2021-12-26 03-06-44

### CODE REVIEW

File: ui/src/components/UI/SyntaxHighlighter/index.tsx [hunk 1]
```diff
@@ -1,30 +1,51 @@
...
+    const markers = [];
+
+    if (showLineNumbers) {
+        for (var i = 0; i < arr.length; i++) {
+            markers.push({
+                line: i + 1,
+                className: 'hljs-marker-line'
+            });
+        }
+    }
```
Comment (lirazyehezkel) [inline ui/src/components/UI/SyntaxHighlighter/index.tsx:47]: I would replace lines 37-47 with this, wdyt?
```
const arr = code.split("\n");
const markers = showLineNumbers ? arr.map((item, i) => {return {
                line: i + 1,
                className: 'hljs-marker-line'
            }}) : [];
```

File: ui/src/components/EntryDetailed/EntrySections.tsx [hunk 2]
```diff
@@ -159,24 +165,26 @@ export const EntryBodySection: React.FC<EntryBodySectionProps> = ({
...
-                <span style={{marginLeft: '.5rem'}}>Wrap text</span>
+                <span style={{marginLeft: '.2rem'}}>Line numbers</span>
+
+                {isBase64Encoding && <div style={{paddingTop: 3, paddingLeft: 20}}>
+                    <Checkbox checked={decodeBase64} onToggle={() => {setDecodeBase64(!decodeBase64)}}/>
+                </div>}
+                {isBase64Encoding && <span style={{marginLeft: '.2rem'}}>Decode Base64</span>}
             </div>
 
             <SyntaxHighlighter
```
Comment (lirazyehezkel) [inline ui/src/components/EntryDetailed/EntrySections.tsx:185]: Where do you provide the language?

File: ui/src/components/UI/SyntaxHighlighter/index.tsx (hunk 1 above)
Comment (mertyildiran) [reply ui/src/components/UI/SyntaxHighlighter/index.tsx]: @lirazyehezkel done with https://github.com/up9inc/mizu/commit/e2299c831c009565e476c5be3fdeb62006535091

File: ui/src/components/EntryDetailed/EntrySections.tsx (hunk 2 above)
Comment (mertyildiran) [reply ui/src/components/EntryDetailed/EntrySections.tsx:192]: @lirazyehezkel the language is detected among [this subset of languages](https://github.com/up9inc/mizu/blob/fix/body-highlight-prettify/ui/src/components/UI/SyntaxHighlighter/index.tsx#L16-L24).

**Review by lirazyehezkel — APPROVED** [review]

**Pull request merged**